aws/resource_aws_dx_connection.go

```diff
@@ -43,6 +43,10 @@ func resourceAwsDxConnection() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"jumbo_frame_capable": {
```

Comment:
See comment on attribute naming: https://github.com/terraform-providers/terraform-provider-aws/pull/6141/files#r224957129.

Reply:
I'm going to go with `jumbo_frame_capable` to match the API.  I'll make this update to the other PRs where they don't match this convention.